firebase-project/src/views/postList.vue

```diff
@@ -32,23 +45,92 @@
               {{ post.description }}
             </p>
 
-            <p v-if="post.taggedUser" class="mt-2 text-xs text-gray-700">
+            <p
```

Comment:
@mukeshsin can you create a component for this one too? 
Searching the user, showing the user, tagging. All in 1 component